### PR TITLE
Add powerplants

### PR DESCRIPTION
**Pull request by matejpiro**

### CODE REVIEW

**Comment by jankrcal** [comment]
What's the status of this PR. Is it ready for review?

**Comment by matejpiro** [comment]
Not yet:(. Still working on this. Sorry.

File: notebooks/regional_emissions_by_category_lib/emission_plots.py [hunk 1]
```diff
@@ -30,26 +32,70 @@ def _get_sum(keys, df):
...
+def _create_powerplant_allowances_list(definition):
+    """Creates list of powerplants with allowances from definition"""
+    allowances = []
+
+    for inner_cat in definition:
+        if "breakdown" not in inner_cat:
+            continue
+        for outer_cat in inner_cat["breakdown"]:
+            if 'allowances' in outer_cat:
+                for item in (outer_cat["sum"]):
```
Comment (jankrcal) [inline notebooks/regional_emissions_by_category_lib/emission_plots.py:70]: minor: you can simplify code and avoid this for cycle by writing directly:
allowances += outer_cat["sum"]

File: notebooks/regional_emissions_by_category_lib/emission_plots.py [hunk 2]
```diff
@@ -30,26 +32,70 @@ def _get_sum(keys, df):
...
+        if 'reminder' in wedge_def:
             wedge_value = _get_value(total_value_code, df) - cumulative_sum
-        perc_dict[wedge_code] = (
-                    wedge_value / total_divider)
-        df.loc[wedge_code, 'value'] = wedge_value  # add a new line to current df
+            _add_line_to_df(wedge_def['code'], wedge_value, df)
+
+    # third pass to compute values for outer perc dict (if in second pass, total outer chart sum > 1 so plot gets error)
+    for wedge_def in definition:
+        wedge_value = _get_value(wedge_def['code'], df)
```
Comment (jankrcal) [inline notebooks/regional_emissions_by_category_lib/emission_plots.py:94]: minor: it seems you could merge this third pass with the second pass (after computing and storing the reminder).

File: notebooks/regional-emissions-by-category-CZ.ipynb [hunk 3]
```diff
@@ -18,9 +18,49 @@
     "              'color': '#c22628',\n",
```
Comment (jankrcal) [inline notebooks/regional-emissions-by-category-CZ.ipynb:18]: Could you please fix an existing bug here (and in the SK plot) by adding a wedge for fugitive emissions (CRF1B, "Úniky při těžbě paliv") into the energy sector?

Exactly the same way it is in the EU plot.

File: notebooks/regional-emissions-by-category-CZ.ipynb [hunk 4]
```diff
@@ -18,9 +18,49 @@
...
     "              'sum': ['CRF1A1C', 'CRF1A1B']},\n",
```
Comment (jankrcal) [inline notebooks/regional-emissions-by-category-CZ.ipynb:19]: Could we please unify this label across all three plots? I'd lean towards shorter "Výroba a úprava paliv".

**Comment by jankrcal** [comment]
This looks great, thanks!

I have a few minor comments, once you address them, please merge!

**Review by jankrcal — APPROVED** [review]
approved once addressing comments

File: notebooks/regional_emissions_by_category_lib/emission_plots.py (hunk 1 above)
Comment (matejpiro) [reply notebooks/regional_emissions_by_category_lib/emission_plots.py:70]: right. fixed.

File: notebooks/regional-emissions-by-category-CZ.ipynb (hunk 4 above)
Comment (matejpiro) [reply notebooks/regional-emissions-by-category-CZ.ipynb:19]: right. fixed

File: notebooks/regional_emissions_by_category_lib/emission_plots.py (hunk 2 above)
Comment (matejpiro) [reply notebooks/regional_emissions_by_category_lib/emission_plots.py:94]: If its not an important request I would rather stick to this way. Its a little more undertandable for me.

Comment (matejpiro) [reply notebooks/regional_emissions_by_category_lib/emission_plots.py:94]: ok

File: notebooks/regional-emissions-by-category-CZ.ipynb (hunk 3 above)
Comment (matejpiro) [reply notebooks/regional-emissions-by-category-CZ.ipynb:18]: Sure. Added as a second field in energy sector (if wanted elsewhere let me know).

**Comment by matejpiro** [comment]
Comments from 8.3. addressed.

**Pull request merged**